### PR TITLE
Wait for stopper's tasks to finish to prevent flaky TestGetTruncatableIndexes

### PR DESCRIPTION
**Pull request by dt**
If the processLoop beats the DrainQueue call to popping an event, DrainQueue may return
before the event is actually processed and the test can fail.

Waiting for the stopper to finish tasks before proceeding ensures that if this happens, the event
is still processed before the test proceeds.

[Review on Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/3746)

### CODE REVIEW

File: storage/raft_log_queue_test.go [hunk 1]
```diff
@@ -88,6 +88,7 @@ func TestGetTruncatableIndexes(t *testing.T) {
 	// Enable the raft log scanner and and force a truncation.
 	store.DisableRaftLogQueue(false)
 	store.ForceRaftLogScanAndProcess()
+	stopper.Quiesce()
```
Comment (tamird) [inline storage/raft_log_queue_test.go]: can you put a comment above this line explaining why it's done? This is only the second location in the codebase (outside of stopper.go) that this method is called.

**Comment by dt** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[storage/raft_log_queue_test.go, line 91 \[r2\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/3746#-K7lwbHK4JDIhTCZw5lu-r2-91:-K7m1JI27rc5rlwnhLZa:-810850699) ([raw file](https://github.com/cockroachdb/cockroach/blob/27ec9e4730c9a48d84ebc7ab392db2eed89dd7d9/storage/raft_log_queue_test.go#L91)):*
Done.

I wonder if DrainQueue, or the Force* helper should just do this (though would require plumbing the stopper though).

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/3746)*

**Comment by tamird** [comment]
I'd squash the revert into the "fixing" commit, I don't see the value of keeping them separate.

---

Reviewed 1 of 1 files at r1, 1 of 1 files at r2, 1 of 1 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/3746#-:-K7m2s0CnytdsimLRXLm:-1791774931)*

**Comment by dt** [comment]
Done.

---

Review status: all files reviewed at latest revision, all discussions resolved, some commit checks pending.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/3746#-:-K7m3VjWVedIQBUs3pqe:-499146009)*

**Pull request merged**